gulp-tasks/nodemon.js

```diff
@@ -1,7 +1,18 @@
 var gulp = require('gulp');
 var nodemon = require('gulp-nodemon');
+var env = require('gulp-env');
 
 gulp.task('nodemon', function() {
+  const PROD_PORT = 8080;
+  const DEV_PORT = 8081;
+  // This results in different ports for prod and dev
+  var port = (GLOBAL.config.env === 'prod') ? PROD_PORT : DEV_PORT;
```

Comment:
Well this only applies to the gulp build. If heroku where to start the server, it would be starting node app.js presumably on a prebuilt version of the front end.

From this app.js would use process.env.PORT - i.e. what ever heroku set.

The scenario I haven't accounted for is if the process.env.PORT is defined although running node app.js will select a random port (or so it seems).